### ISSUE TITLE
are you not supporting the plugin anymore?

### ISSUE THREAD

**Issue by zhuker** [Kotlin/kotlin-frontend-plugin#151]

**Comment by joffrey-bion** [comment]
I'm wondering that same thing. It looks like it's abandoned, and yet this is the most official thing we can get for now :'(

**Comment by nimloth05** [comment]
Agree. I think the whole JS Kotlin tooling is barely maintained. I'm in the process of replacing this plugin with a hand written toolchain. I use the com.moowork.node plugin to interact with node.

**Comment by krzema12** [comment]
Some data point from my side: the immaturity of Kotlin JS and this plugin were key when deciding whether to use Kotlin in my new web project. I decided to go with TypeScript because I don't have resources to maintain a custom build toolchain, like @nimloth05 proposed. BTW, @nimloth05, do you plan to share your toolchain? Maybe it will be a good alternative to this plugin.

**Comment by nimloth05** [comment]
@krzema12 I've created an example workspace with various ways to build a kotlin2js app: https://github.com/nimloth05/kotlin2js-examples ATM the custom tooling karma tests are not working

**Comment by krzema12** [comment]
Thanks! 

For the record, JetBrains just announced Kotlin 1.3.40 and that this plugin will be deprecated. The new plugin is available here:https://plugins.gradle.org/plugin/org.jetbrains.kotlin.js
Full announcement: https://blog.jetbrains.com/kotlin/2019/06/kotlin-1-3-40-released/

**Comment by neworld** [comment]
Someone should update the description of this repo about deprecation.
